### DISCUSSION TITLE
Error creating a new page

### DISCUSSION THREAD

**Discussion by MrTimotys** [requarks/wiki#5678] [Error / Bug Report] [answered]
Hello, I'm new to wiki.js. I want to create a new page as a home page, but it always gives me an error and I don't know what to do with it.

`Error when running job render-page: EACCES: permission denied, open '/var/www/wiki/data/cache/fe1082dc88339775ff88bb4a46f2596d8f8aff2a.bin'`

**Comment by NGPixel** [comment, marked as answer]
Make sure your `data` folder is writeable by the wiki.js node process.
